### PR TITLE
psp-4182 allow clicking on a properties boundary in addition to its m…

### PR DESCRIPTION
**Pull request by devinleighsmith**
…arker.

### CODE REVIEW

File: frontend/src/features/properties/map/MapView.tsx [hunk 1]
```diff
@@ -42,8 +42,10 @@ const MapView: React.FC<MapViewProps> = (props: MapViewProps) => {
     history.push(`/mapview/sidebar/property/${property.id}?pid=${property.pid}`);
   };
 
-  const onPropertyViewClicked = (pid?: string | null) => {
-    if (pid !== undefined && pid !== null) {
+  const onPropertyViewClicked = (pid?: string | null, id?: number) => {
+    if (id !== undefined) {
+      history.push(`/mapview/sidebar/property/${id}?pid=${pid}`);
```
Comment (FuriousLlama) [inline frontend/src/features/properties/map/MapView.tsx:47]: pid is not guaranteed to be non null here

Comment (devinleighsmith) [reply frontend/src/features/properties/map/MapView.tsx:47]: having a null pid works here, I've tested it. it just results in the ltsa screen not being shown.

File: geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  </namespace>
+  <title>PIMS_PROPERTY_BOUNDARY_VW</title>
+  <keywords>
+    <string>features</string>
+    <string>PIMS_PROPERTY_BOUNDARY_VW</string>
+  </keywords>
+  <nativeCRS class="projected">PROJCS[&quot;NAD83 / BC Albers&quot;, 
+  GEOGCS[&quot;NAD83&quot;, 
+    DATUM[&quot;North American Datum 1983&quot;, 
+      SPHEROID[&quot;GRS 1980&quot;, 6378137.0, 298.257222101, AUTHORITY[&quot;EPSG&quot;,&quot;7019&quot;]],
```
Comment (asanchezr) [inline geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml:16]: Thanks for adding the new map layer to our local geoserver. Could you please also update **make env** to match these changes?

File: frontend/src/features/properties/map/MapView.tsx (hunk 1 above)
Comment (devinleighsmith) [reply frontend/src/features/properties/map/MapView.tsx:47]: the url ends up with pid=

File: geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml (hunk 2 above)
Comment (devinleighsmith) [reply geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml:16]: @asanchezr my understanding of make .env is that is creates the datastore.xml files needed to connect to a database. In this case, the same datastore previously used for the location view can also be used for the boundary view (note the co-location of both views under the same parent folder).

**Review by FuriousLlama — APPROVED** [review]

File: geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml (hunk 2 above)
Comment (asanchezr) [reply geoserver/geoserver_data/psp/mssql/PIMS_PROPERTY_BOUNDARY_VW/featuretype.xml:16]: Right. I missed that detail. Thanks for clarifying

**Review by asanchezr — APPROVED** [review]

**Pull request merged**